docs/custom_theme/orion_logo.svg

```diff
@@ -0,0 +1,33 @@
+<svg width="80" height="40" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Does this need to be updated to the EthSigner logo?

Reply:
Good catch, it just have to be removed.